app/views/pages/privacy_policy.html.erb

```diff
@@ -178,7 +151,7 @@
       We also use Google’s G Suite to process some personal data. Google processes your data according to our instructions.
     </p>
 
-    <h4 class="govuk-heading-s">Hosting services</h4>
+    <h3 class="govuk-heading-s">Hosting services</h3>
```

Comment:
I think this should be a `govuk-heading-m` as it's smaller than the equally-levelled heading above ☝🏽

Reply:
thank you <3